docs/TSCONFIG.md

```diff
@@ -0,0 +1,150 @@
+# TsConfig files structure
+
+The presence of a config file in a directory indicates that it is the root of a TypeScript project.
+In Stark, several types of tsconfig files are placed in the packages for different purposes, described below.
+
+## tsconfig.json
+
+The `tsconfig.json` file can be found at the following locations:
+
+```txt
++---starter
+|	tsconfig.json
++---showcase
+|	tsconfig.json
++---packages
+	tsconfig.json
+	+---stark-build
+	|	tsconfig.json
+...
+```
+
+The structure of this file is like the following:
+
+```
+{
+	...
+  "extends": "./node_modules/@nationalbankbelgium/stark-build/tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "outDir": "./dist",
+    ...
+    }
+  },
+  "exclude": ["node_modules", "dist"]
+  ...
+}
+```
+
+This file is used simply to build the packages where it is located.
```

Comment:
Maybe you can elaborate a bit more on this becase we have a special case which is `packages/tsconfig.json`. That file is only used in our build process.